minimisers/simplex/fMinSearch.m

```diff
@@ -257,13 +272,15 @@
     % xbar = average of the n (NOT n+1) best points
     xbar = sum(v(:,1:n), 2)/n;
     xr = (1 + rho)*xbar - rho*v(:,end);
-    x(:) = xr; [fxr, result] = funfcn(x,varargin{:});
+    x(:) = xr; [fxr, result] = funfcn(x, problemStruct, controls, params);
     func_evals = func_evals+1;
     
     if fxr < fv(:,1)
         % Calculate the expansion point
         xe = (1 + rho*chi)*xbar - rho*chi*v(:,end);
-        x(:) = xe; [fxe, result] = funfcn(x,varargin{:});
+
+        x(:) = xe; [fxe, result] = funfcn(x, problemStruct, controls, params);
+        controls.calcSLD = false;
```

Comment:
Is this in the right place? There are other evaluations of `funfcn` in the `else` clause of the `if` statement on line 278 that are not followed by `controls.calcSLD = false;`

Reply:
I will remove as this is already handled by line 353. I probably added this when I hoped to decipher the else branch